examples/seismic/tutorials/16_ader_fd.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "31226b14-e907-4503-9918-a09163363217",
```

Comment:
<p>Choose one you derive in the math and leave the rest outII</p><p> </p><p>Something needs to be said about ABCs. It is a bit non intuitive and potentially dangerous that it doesn't need any so there needs to be some explanation why</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2450/discussion/#comment-1758172539'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Added an explanation. Essentially the discretisation introduces numerical diffusion where the solution is non-smooth, so when the wave hits the zero padding, it will diffuse outwards (although the zeroes never get updated).